docs/reference/access-control-lists.md

```diff
@@ -1,37 +1,52 @@
-| **Write**  | Allows all data read and write operations. | Developers                | FSReadWriteAll \[fs:ListRepositories, fs:ReadRepository, fs:ReadCommit, fs:ListBranches, fs:ListTags, fs:ListObjects, fs:ReadObject, fs:WriteObject, fs:DeleteObject, fs:RevertBranch, fs:ReadBranch, fs:ReadTag, fs:CreateBranch, fs:CreateTag, fs:DeleteBranch, fs:DeleteTag, fs:CreateCommit] RepoManagementReadAll \[ci:Read*, retention:Get*, branches:Get*, fs:ReadConfig] |
-| **Super**  | Allows all operations except auth.         | SuperUsers (with changes) | FSFullAccess  \[fs:*] RepoManagementReadAll \[ci:Read*, retention:Get*, branches:Get*, fs:ReadConfig]                                                                                                                                                                                                                                                                            |
-| **Admin**  | Allows all operations.                     | Admins                    | AuthFullAccess \[auth:*]  FSFullAccess \[fs:*]  RepoManagementFullAccess \[ci:*, retention:*, branches:*, fs:ReadConfig]                                                                                                                                                                                                                                                         |
 
-### Scopes
+## Scopes
 
 When granted to a group, permissions Read, Write, and Super may be scoped to a set of repositories.
+
 Admin includes global abilities that apply across repos and cannot be scoped to a set of repositories
 
-### ACL
 
-You can now attach Permissions and scope them to groups in the Groups page
-There are 4 default groups, named after the 4 permissions. Each group is global (applies for all repositories).
-The group list initially would be
+## Plugable Authentication and Authorization
 
-| Group ID    | Default Permission | Created at       | Repositories |
-|-------------|--------------------|------------------|--------------|
-| **Read**    | Read               | \<Creation date> | All          |
-| **Write**   | Write              | \<Creation date> | All          |
-| **Super**   | Super              | \<Creation date> | All          |
-| **Admin**   | Admin              | \<Creation date> | All          |
+Authorization and authentication is pluggable in lakeFS. If lakeFS is attached to an external authorization server, the existing RBAC GUI may continue to be used. It is not possible to use both types of GUI at the same time: Moving from RBAC to simplified may only be performed once and **will** lose configuration.
```

Comment:
as well as RBAC GUI, presumably `lakectl` too? 

and when it says "use both types of GUI" what's the other type of GUI? Is there a GUI for new ACLs, and if so is there a doc that covers it?

Reply:
There is only one GUI.  `lakectl` might be considered another UI.  I agree that it is confusing.  Perhaps just document that we now support `auth.ui_config.RBAC` with values `simplified` and `external`, and that that influences the UI presented.  (As the name implies, `external` requires an external auth server.  lakeFS.Cloud provides such a server.)

Reply:
@arielshaqed I'm slowly getting up to speed here :) I've pushed a change that reflects my understanding now. LMK if it looks right, if not perhaps we get on a call to go through it. thanks.